server/src/main/java/com/scalar/db/server/AdminService.java

```diff
@@ -66,4 +67,11 @@ public void stats(Empty request, StreamObserver<StatsResponse> responseObserver)
     responseObserver.onNext(StatsResponse.newBuilder().setStats("{}").build());
     responseObserver.onCompleted();
   }
+
+  @Override
+  public void checkPaused(Empty request, StreamObserver<CheckPausedResponse> responseObserver) {
+    responseObserver.onNext(
+        CheckPausedResponse.newBuilder().setPaused(!gateKeeper.letIn()).build());
```

Comment:
use `Gatekeeper.letIn()` as the current paused status

Reply:
I don't think we can use `letIn()` to get the current paused status.

Maybe we need to add something like `isOpen()` method to `GateKeeper`, and make the new method return the current paused status (the `isOpen` field in `LockFreeGateKeeper` and `SynchronizedGateKeeper`).

Reply:
Understood.
Let me revise it.

Reply:
@brfrn169 
Sorry for my mistake.
I add `isOpen` method in https://github.com/scalar-labs/scalardb/pull/583/commits/acb118db9e08acc1cdd87c55946549363d91d75d

Please take a look!
Thank you 😄